### PR TITLE
updated mongodb image version and replication section

### PR DESCRIPTION
**Pull request by gaurav-nelson**
For https://bugzilla.redhat.com/show_bug.cgi?id=1546724

- mongodb `3.4` image updates
- Replication section rewrite
- minor formatting fixes

Preview: https://bug1546724-fixes--gnelson-preview.netlify.com/openshift-enterprise/(head%20detached%20at%20fetch_head)/using_images/db_images/mongodb.html#using-mongodb-replication

### CODE REVIEW

**Comment by gaurav-nelson** [comment]
@openshift/team-documentation PTAL

File: using_images/db_images/mongodb.adoc [hunk 1]
```diff
@@ -373,251 +373,185 @@ endif::[]
...
-To xref:../../dev_guide/deployments/basic_deployment_operations.adoc#scaling[increase the number of members]
-in the cluster:
+To scale down:
 
+. Set the new number of replicas by using the `oc scale` command:
++
+[source, bash]
 ----
-$ oc scale rc mongodb-1 --replicas=<number>
+$ oc scale --replicas=3 petset/mongodb
```
Comment (mburke5678) [inline using_images/db_images/mongodb.adoc]: Do we need to define `petset/mongodb`?

File: using_images/db_images/mongodb.adoc [hunk 2]
```diff
@@ -373,251 +373,185 @@ endif::[]
...
-=== Creating the Deployment Configuration
-
-To set up MongoDB replication, a
-xref:../../architecture/core_concepts/deployments.adoc#deployments-and-deployment-configurations[deployment
-configuration] is defined in the example template that defines a
-xref:../../architecture/core_concepts/deployments.adoc#replication-controllers[replication
-controller]. The replication controller manages the members of the MongoDB
-cluster.
+While a replica set member is down or being restarted, it may be in one of these
+scenarios:
```
Comment (mburke5678) [inline using_images/db_images/mongodb.adoc]: s/scenarios/states ??

**Comment by mburke5678** [comment]
@gaurav-nelson A few comments. Otherwise LGTM

**Pull request merged**